performance/sqlserver/backup_maintenance_planning.rst

```diff
@@ -0,0 +1,8 @@
+Backup maintenance plan
+=======================
+
+This is really outside the scope of Firely Server, but when using FS on SQL Server needyou to have a proper maintenance plan in place and regularly back-up the transaction log when when the recovery model is set to anything but Simple.
```

Comment:
Did you mean something along the lines of: 'In case Firely Server is used in conjunction with SQL Server, a maintenance plan should be in place and regular back-ups of the transaction logs should be made if the recovery log is not set to 'Simple'.'

Reply:
Yes